WORKSPACE

```diff
@@ -1,22 +1,30 @@
 workspace(name = "bazel_skylib")
 
-load(":workspace.bzl", "bazel_skylib_workspace")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "bazel_federation",
+    url = "https://github.com/bazelbuild/bazel-federation/archive/130c84ec6d60f31b711400e8445a8d0d4a2b5de8.zip",
+    sha256 = "9d4fdf7cc533af0b50f7dd8e58bea85df3b4454b7ae00056d7090eb98e3515cc",
+    strip_prefix = "bazel-federation-130c84ec6d60f31b711400e8445a8d0d4a2b5de8",
+    type = "zip",
+)
+
+load("@bazel_federation//:repositories.bzl", "bazel_skylib_deps")
+
+bazel_skylib_deps()
 
-bazel_skylib_workspace()
+load("@bazel_federation//setup:bazel_skylib.bzl", "bazel_skylib_setup")
+
+bazel_skylib_setup()
```

Comment:
Why do we need to call this function?

Reply:
bazel_skylib_setup() is basically a wrapper around bazel_skylib_workspace. It would call more functions if we added dependencies that also needed setup.

We can keep the call to bazel_skylib_workspace(), but I thought it would be nicer (and future-proof) to use the standard federation structure here.